go-controller/pkg/ovn/loadbalancer.go

```diff
@@ -187,15 +188,21 @@ func (ovn *Controller) createLoadBalancerRejectACL(lb string, sourceIP string, s
 	}
 	vip := util.JoinHostPortInt32(sourceIP, sourcePort)
 	// NOTE: doesn't use vip, to avoid having brackets in the name with IPv6
+	// NOTE: ACL names are limited to 63 characters and is limited in the cmd creation
 	aclName := fmt.Sprintf("%s-%s:%d", lb, sourceIP, sourcePort)
+	//aclName = strings.ReplaceAll(aclName, ":", "\\:")
+	//aclName = aclName[:63]
+	if len(aclName) > 63 {
+		klog.Infof("BILLY: Limiting ACL Name from %s to %.63s to keep under 63 characters", aclName, aclName)
+	}
 	// If ovn-k8s was restarted, we lost the cache, and an ACL may already exist in OVN. In that case we need to check
 	// using ACL name
 	aclUUID, stderr, err := util.RunOVNNbctl("--data=bare", "--no-heading", "--columns=_uuid", "find", "acl",
-		fmt.Sprintf("name=%s", strings.ReplaceAll(aclName, ":", "\\:")))
+		fmt.Sprintf("name=%.63s", strings.ReplaceAll(aclName, ":", "\\:")))
```

Comment:
We can't just do this; the end of the `aclName` is important. You might create duplicate names by truncating.

(This seems like an unrelated bugfix anyway? File a new issue/PR for this?)